lib/api/list-actions.js

```diff
@@ -1,19 +1,37 @@
+const debug = require('debug')('codepress:codeceptjs-factory');
+const fs = require('fs');
+const path = require('path');
 const codeceptjsFactory = require('../model/codeceptjs-factory');
 const { container } = codeceptjsFactory.getInstance();
 const { getParamsToString } = require('codeceptjs/lib/parser');
 const { methodsOfObject } = require('codeceptjs/lib/utils');
 
 module.exports = (req, res) => {
+  const docsWebApiFolderPath = path.join(__dirname, '../../node_modules/codeceptjs/docs/webapi');
+  const docFileList = [];
+  fs.readdirSync(docsWebApiFolderPath).map(fileName => {
+    docFileList.push(fileName.split('.')[0]);
```

Comment:
I think you can use `fs.baseName` or similar function here

Reply:
yes, let me change this part of the code.

Reply:
Did the required change.